16.0 mig pos access right

jumeldi74 opened the pull request.

legalsylvain committed:
[ADD] new module 'pos_access_right'

legalsylvain committed:
[REF] improve description;

legalsylvain committed:
[ADD] fr translation;

legalsylvain committed:
[ADD] new blockage;

legalsylvain committed:
[ADD] fr translation;
[IMP] description;

legalsylvain committed:
[FIX]

legalsylvain committed:
[FIX] display;

legalsylvain committed:
[FIX] remove useless feature;[FIX] bug on multi order check

legalsylvain committed:
[REF] rename fields

legalsylvain committed:
[FIX] add missing author

legalsylvain committed:
[REF] improve fr translation

oca-transbot committed:
OCA Transbot updated translations from Transifex

oca-travis committed:
[UPD] Update pos_access_right.pot

Commit:
[MIG] pos_access_right: Migration to 12.0

[DOC] update README

[FIX] fix variable name

[REF] the pos.config model load all fields

[DOC] update website link

[LINT] fix lint errors

[LINT] fix lint errors

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update pos_access_right.pot

oca-transbot committed:
Update translation files

Updated by Update PO files to match POT (msgmerge) hook in Weblate.

Commit:
Added translation using Weblate (Russian)

Commit:
Translated using Weblate (Russian)

Currently translated at 50.0% (10 of 20 strings)

Translation: pos-12.0/pos-12.0-pos_access_right
Translate-URL: https://translation.odoo-community.org/projects/pos-12-0/pos-12-0-pos_access_right/ru/

KolushovAlexandr committed:
[ADD] block payment button

oca-travis committed:
[UPD] Update pos_access_right.pot

legalsylvain committed:
[FIX] eslint

oca-travis committed:
[UPD] Update pos_access_right.pot

OCA-git-bot committed:
pos_access_right 12.0.1.0.1

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: pos-12.0/pos-12.0-pos_access_right
Translate-URL: https://translation.odoo-community.org/projects/pos-12-0/pos-12-0-pos_access_right/

OCA-git-bot committed:
[UPD] README.rst

ammarofficewla committed:
[MIG] pos_access_right : Migaration to 13.0

hkapatel-initos committed:
[IMP] : black, isort, prettier

hkapatel-initos committed:
[MIG] pos_access_right: Migration to 14.0

oca-travis committed:
[UPD] Update pos_access_right.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
pos_access_right 14.0.1.0.1

kh-badep committed:
Translated using Weblate (French)

Currently translated at 100.0% (23 of 23 strings)

Translation: pos-14.0/pos-14.0-pos_access_right
Translate-URL: https://translation.odoo-community.org/projects/pos-14-0/pos-14-0-pos_access_right/fr/

Noel000 committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (23 of 23 strings)

Translation: pos-14.0/pos-14.0-pos_access_right
Translate-URL: https://translation.odoo-community.org/projects/pos-14-0/pos-14-0-pos_access_right/ca/

legalsylvain committed:
[FIX] pos_access_right : remove bad css that impact all PoS

OCA-git-bot committed:
pos_access_right 14.0.1.0.2

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (25 of 25 strings)

Translation: pos-14.0/pos-14.0-pos_access_right
Translate-URL: https://translation.odoo-community.org/projects/pos-14-0/pos-14-0-pos_access_right/it/

jumeldi74 committed:
[IMP] pos_access_right: pre-commit stuff

jumeldi74 committed:
[MIG] pos_access_right: Migration to 16.0

hitrosol commented:
Thanks @jumeldi74 for the PR. 
Not sure why the checking is only for runboat/build.

IMO this PR can improve the previous one https://github.com/OCA/pos/pull/957

jumeldi74 committed:
[MIG] pos_access_right: Migration to 16.0 - revision

[MIG] pos_access_right: Migration to 16.0 - revision 2

[MIG] pos_access_right: Migration to 16.0 - adjusted test unit and add one line information in description.

[MIG] pos_access_right: Migration to 16.0 - repair test unit error.

[MIG] pos_access_right: Migration to 16.0 - still repair test unit error.

[MIG] pos_access_right: Migration to 16.0 - give access right for backspace button in keyboard and button remove in order line to delete order line.

jumeldi74 commented:
@legalsylvain Please review my PR. Thank You

jumeldi74 committed:
[MIG] pos_access_right: Migration to 16.0 - final modification and merging.

jumeldi74 committed:
[MIG] pos_access_right: Migration to 16.0 - Adding Refund Action Access Right.

jumeldi74 committed:
[MIG] pos_access_right: Migration to 16.0 - Add access right for Cash In/Out Button only for Accounting/Manager.

jumeldi74 committed:
[MIG] pos_access_right: Migration to 16.0 - Add group access right in Cash In/Out Button only for Accounting/Manager (Revision).

jumeldi74 committed:
[MIG] pos_access_right: Migration to 16.0 - Add group access right in Cash In/Out Button only for Accounting/Manager (Revision 2).
